fapolicy_analyzer/ui/policy_rules_admin_page.py

```diff
@@ -324,34 +324,35 @@ def on_next_system(self, system):
         )
 
     def on_user_selection_changed(self, secondaryAction, data):
-        uid = data[1] if data else None
-        if uid != self.selectedUser:
-            self.selectedUser = uid
-            self.selectedGroup = None
-            self.__populate_acl_details(uid, acl.getUserDetails)
-            secondaryAction()
+        uids = [datum[1] for datum in data] if data else [None]
+        for uid in uids:
+            if uid != self.selectedUser:
+                self.selectedUser = uid
+                self.selectedGroup = None
+                self.__populate_acl_details(uid, acl.getUserDetails)
+                secondaryAction()
```

Comment:
Much like on the ATDA, you are wasting cycles by looping over the list here.  We are only displaying the last selected users data.  In fact allowing multiselect on the users and groups is pointless as it is right now.  The user can only act on the last selected user.  Are there plans to merge the lists from multi users/groups for the secondary action?